code/game/turfs/simulated/floor.dm

```diff
@@ -54,6 +58,12 @@ var/list/wood_icons = list("wood","wood-broken")
 	else
 		icon_regular_floor = icon_state
 
+/turf/simulated/floor/Destroy()
+	//No longer phazon, not a teleport destination
+	if(material=="phazon")
```

Comment:
I hate this implementation but it's not your fault, floors are absolutely ghastly, horrifying.